packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/CustomUserControlViewManagerCPP.cpp

```diff
@@ -105,18 +105,18 @@ void CustomUserControlViewManagerCpp::UpdateProperties(
 }
 
 // IViewManagerWithCommands
-IMapView<hstring, int64_t> CustomUserControlViewManagerCpp::Commands() noexcept {
-  auto commands = winrt::single_threaded_map<hstring, int64_t>();
-  commands.Insert(L"CustomCommand", 0);
+IVectorView<hstring> CustomUserControlViewManagerCpp::Commands() noexcept {
+  auto commands = winrt::single_threaded_vector<hstring>();
+  commands.Append(L"CustomCommand");
   return commands.GetView();
 }
 
 void CustomUserControlViewManagerCpp::DispatchCommand(
```

Comment:
since we are going to support both the int and string variants for the time being, should this be an overload rather than changing it to string so we can test both cases?

Reply:
I touched on that a little in the PR description. Internal CPP code needs to support both, but the C++ view manager exposing IDs to implementers is a departure from the C# and ObjC view managers at least.

This has some motivation to move the direction of only strings, but the bigger concern I had was around API confusion. I.e. you look at an interface and see a command id version and a string version. It's super opaque to know which to implement when.